.travis.yml

```diff
@@ -13,7 +13,8 @@ script:
   - npm run check:themes
   - npm run lint
   - npm run test:unit -- --coverage
-  - '[[ ! -z "$SAUCE_ACCESS_KEY" ]] && travis_wait npm run test:visual || true'
+  - '[[ ! -z "$SAUCE_ACCESS_KEY" ]] && npm run docs || true'
+  - '[[ ! -z "$SAUCE_ACCESS_KEY" ]] && npm run test:visual || true'
```

Comment:
Am I right in thinking this will never fail due to the `|| true`?

Reply:
I'm not sure. It may just be a short hand way if ending the if? In any case, probably needs changing.

Reply:
This has been addressed in https://github.com/stencila/thema/pull/126/commits/cbcf77e34c1960a41f8ce3d7fa430b01d3cd35ca